metagraph/src/graph/annotated_dbg.hpp

```diff
@@ -80,6 +80,8 @@ class AnnotatedDBG : public AnnotatedSequenceGraph {
 
     bool check_compatibility() const;
 
+    std::vector<node_index> sequence_to_path(std::string_view sequence) const;
```

Comment:
Since this calls `map_to_nodes`, it doesn't exactly map to paths. How about `sequence_to_labeled_nodes`?

Reply:
renamed to `sequence_to_lnodes`